docs/packaging.md

```diff
@@ -87,25 +89,30 @@ The `iotedge` CLI tool is used to interact with the `aziot-edged` service.
 
 ```
 /etc/aziot/
+├── config.toml.template
 ├── certd/
-│   └── config.toml
+│   └── config.toml.default
 ├── identityd/
-│   └── config.toml
-└── keyd/
-    └── config.toml
+│   └── config.toml.default
+├── keyd/
+│   └── config.toml.default
+└── tpmd/
+    └── config.toml.default
 ```
 
-Note that the configuration is now in TOML format.
+Note that the configuration is now in TOML format. The `aziotctl config apply` command takes a configuration file in TOML format as input and generates the Individual `config.toml` file for each service. The provided `config.toml.template` is a template for configuration values that can be set. It can be used as the super `config.toml` from which the others are generated.
+
 </td>
 <td>
 
 ```
 /etc/aziot/
+├── config.toml.edge.template
 └── edged/
-    └── config.yaml
+    └── config.toml
 ```
 
-Note that the configuration is still in YAML format, but reduced in scope.
+The `config.toml.edge.template` is a template for a super `config.toml` that additionally includes configuration values specific to IoT Edge. The role of that super `config.toml` replaces the `config.yaml` that was previously used. It will be common to use a copy of the template as the super `/etc/aziot/config.toml`. The `iotedge config apply` wraps `aziotctl config apply` in addition to generating the individual `config.toml` for the `aziot-edged` service.
```

Comment:
This paragraph belongs in the iotedge repo. We should only talk about `config.toml.template` here.

Reply:
Why not move this entire document to the iotedge repo?

Reply:
The users of the a-i-s package need documentation for the contents of the a-i-s package.

Reply:
If you're agreed, I'll put a copy of this doc in the iotedge repo with most of the content that is currently here.  And then this copy I'll strip of anything that isn't only for this package which would mean it becomes most of the first paragraph with the middle column of the table (only), and then the very last section about installing a-i-s.  Sound good?

Reply:
re: what will be left in this file, yes. The `aziotctl config wizard` stuff is also wrong and should talk about `aziotctl config apply`, but I'll fix that as part of my PR to remove `wizard`.

re: putting a copy of this in the iotedge repo, whether you want to do that or link to this one when talking about the common parts (to reduce duplication) is up to you.

Reply:
I went ahead and included the fixes you made here in https://github.com/Azure/iot-identity-service/pull/145 along with getting rid of `aziotctl config wizard`. Please review that. For this PR, you can close it and reopen against the iotedge repo.